### ISSUE TITLE
Additional eyebrowse-mode-line-style

### ISSUE THREAD

**Issue by novoid** [wasamasa/eyebrowse#89]
Related to #88 where we do not necessary have to display all the window configs in the modeline, we could also add new `eyebrowse-mode-line-style`.

Suppose, we have the current setup with `[1:org 2:config 3:coding]`:

- "numbers-only"  `[1 2 3]` (with color-highlighting the current one)
- "current-only" `[2:config]`
- "current-number-only" `[2]`

**Comment by wasamasa** [comment]
Write a custom indicator. The Spacemacs people did it, so anyone can, really.

**Comment by novoid** [comment]
You were right. Even I could do it 😉

I did a minimal solution with a copy of `eyebrowse-format-slot()` and `eyebrowse-mode-line-indicator()` which I added to the `define-minor-mode eyebrowse-mode` and suppresses the "tags" (labels/descriptions).

In my own `eyebrowse-format-slot-mini()` I changed
```
         (format-string (if (and tag (> (length tag) 0))
                            eyebrowse-tagged-slot-format
                          eyebrowse-slot-format))
```
to 
```
         (format-string eyebrowse-slot-format) ;; mini: omit the check for a tag
```

"current-only" and "current-number-only" are not addressed with it though.

Works for me now.
